src/components/chatroom/chatlist/chatlist.chatroom.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import Data from "../friends";
+import { Link } from "react-router-dom";
+import "./chatlist.chatroom.scss";
+
+function Preview({ user, name, lastChat, url }) {
+  return (
+    <div className="row">
+      <img src={url} alt={user}></img>
+      <div>
+        <h3>{name}</h3>
+        <p>{lastChat}</p>
+      </div>
+    </div>
+  );
+}
+
+function ChatList({ chats, setChats }) {
+  return (
+    <>
+      <div className="chatlist">
+        {chats.map((object, i) => {
+          return (
+            <Link
+              to={`/chatroom/${object.id}`}
+              style={{ textDecoration: "none", color: "black" }}
+            >
+              <Preview
+                key={i}
+                user={object.id}
+                name={object.name}
+                lastChat={object.chats[object.chats.length - 1].chat}
+                url={object.img}
```

Comment:
```suggestion
                profileImageUrl={object.img}
```
props가 어떤 용도인지 조금더 명확하게...!